UIComponents/Sources/UIComponents/TabBar/TabBarItems/PlainTabBarItemView.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+
+public class PlainTabBarItemView: UIView {
+    // MARK: - Private properties
+    private let model: TabBarItemViewModel
+    private let titleLabel = UILabel()
+    private let imageView: UIImageView = {
+        let view = UIImageView()
+        view.contentMode = .center
+        return view
+    }()
+
+    // MARK: - Initialization
+    public init(model: TabBarItemViewModel) {
+        self.model = model
+        super.init(frame: .zero)
+        setupUI()
+        fillContent()
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension PlainTabBarItemView: TabBarItemView {
+    // MARK: - Public interface
+    public func setSelected(_ isSelected: Bool) {
+        UIView.animate(withDuration: 0.1) {
+            if isSelected {
+                self.imageView.image = self.model.icon?.withTintColor(
+                    self.designEngine.colors.accentTint.uiColor
+                )
+            } else {
+                self.imageView.image = self.model.icon?.withTintColor(
+                    self.designEngine.colors.textPrimary.uiColor
+                )
+            }
+            self.titleLabel.isHidden = !isSelected
```

Comment:
`isHidden` isn't an animatable property. If you want to animate hiding, you should use a discrete property `alpha`.

Reply:
You might use alpha in the animation block, and then use isHidden in the completion.

Reply:
Fixed